mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -28,6 +28,7 @@
 import java.util.Map;
```

Comment:
The integration test does not confirm the even such as 
1/ and 2/ are still OK even without the (Wait) command cause:
Annotation is saved with MailboxId as its key so it needs to look up mailbox by mailboxPath to achieve mailboxId whenever it needs to do anything with Annotation. So:
1/ mailboxId is gone so it can not confirm Annotation even is called or not
2/ new mailboxId is created so cannot confirm the old one is still there

I'm facing the issue with even listener. So if we listen to new even for annotation when deleting mailbox --> it cannot lookup mailboxId after that.